packages/cli/src/commands/deploy/render.js

```diff
@@ -36,29 +34,35 @@ export const builder = (yargs) => {
     )
 }
 
-export const handler = async ({ side, prisma, dm: dataMigrate, serve }) => {
+export const handler = async ({ side, prisma, dm: dataMigrate }) => {
   const paths = getPaths()
   let commandSet = []
-  if (side == 'api') {
+  if (side === 'api') {
     if (prisma) {
       commandSet.push('yarn rw prisma migrate deploy')
     }
     if (dataMigrate) {
       commandSet.push('yarn rw dataMigrate up')
     }
-    if (serve) {
-      commandSet.push('yarn rw serve api')
-    }
-  } else if (side == 'web') {
+  } else if (side === 'web') {
     commandSet.push('yarn')
     commandSet.push('yarn rw build web')
   }
 
-  execa(commandSet.join(' && '), {
-    shell: true,
-    stdio: 'inherit',
-    cwd: paths.base,
-    extendEnv: true,
-    cleanup: true,
-  })
+  if (commandSet.length) {
+    await execa(commandSet.join(' && '), {
+      shell: true,
+      stdio: 'inherit',
+      cwd: paths.base,
+      extendEnv: true,
+      cleanup: true,
+    })
+  }
+
+  if (side === 'api') {
+    await apiServerHandler({
+      port: getConfig().api?.port || 8911,
+      apiRootPath: '/',
+    })
+  }
```

Comment:
@SEANDOUGHTY @crcastle @dac09 
Questions about the RW API server config here:
1. **Port:** does it matter what the port is? And/or whether or not it respects redwood.toml web.port value?
2. **apiRootPath:** I'm assuming this does indeed need to be hardcoded here, correct?
3. **NODE_ENV:** Is the Render service automatically NODE_ENV set? If not, we should probably do a check and set if not exists similar to what we do over [here in this `rw serve` command code snippet](https://github.com/redwoodjs/redwood/blob/70ba72f00828b4ddcd1212b5f40c59af560ea2d1/packages/cli/src/commands/serve.js#L84-L87).